Inventory/RestraintTypes/RestraintRopeHarness.gd

```diff
@@ -30,7 +33,9 @@ func doStruggle(_pc, _minigame):
 		damage = calcDamage(_pc, 0.5)
 		stamina = 10
 		lust = scaleDamage(5)
-	
+	if fatalFail(_minigame):
+		text += " but it seems like you just tightened it up more"
```

Comment:
```suggestion
		text += " but it seems like {user.youHe} just tightened it up more"
```
`user.youHe` translates into `he` or `she` for npcs and into `you` if user is player